source/installation/file-location.txt

```diff
@@ -0,0 +1,74 @@
+.. _file-location:
+
+==================================
+Relational Migrator File Locations
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Review the location of files needed when working with Relational Migrator. 
+Their location varies according to the OS you are working with. 
+
+Mac
+---
+
+The file locations for the Mac OS:
+
+- **Configuration file**
+  
+  ``~/Library/Application Support/MongoDB/Relational Migrator/user.properties``
+
+- **Driver downloads**
+
+  ``~/Library/Application Support/MongoDB/Relational Migrator/Drivers``
```

Comment:
This is incorrect; tilde needs to be removed. This difference is subtle and should be emphasised, i.e. the config and logs are under the user's home folder, while the drivers are shared for all users.

Reply:
Done.